supplementary_style_guide/glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc

```diff
@@ -0,0 +1,762 @@
+[[partner-and-hybrid-cloud-infrastructure-conventions]]
+
+[discrete]
+[[access-mode]]
+==== image:images/yes.png[yes] access mode (noun)
+*Description*: The concurrency method used when accessing a storage component, such as `readwriteonce` or `readwritemany`.
+
+*Use it*: Yes
+
+
+[discrete]
+[[availability]]
+==== image:images/caution.png[with caution] availability (adjective)
+The state of being available to perform work in a system, in the context of high availability or availability zones. The ability of a component to be utilized at a given time.
+
+Ensure that availability is not erroneously used to indicate that a server or disk is online or available, e.g. do not say "the server has availability".
+
+*Use it*: With caution
+
+*See also*: xref:high-availability[high availability], xref:availability-zone[availability zone]
+
+
+[discrete]
+[[availability-zone]]
+==== image:images/yes.png[yes] availability zone (noun)
+*Description*: A logical context or geographical region used to provide high availability. This grouping can be defined by a technology, or according to unique use cases.
+
+Ensure that availability is not erroneously used to indicate that a server or disk is online or available, e.g. do not say "the server has availability".
+
+*Use it*: Yes
+
+*See also*: xref:high-availability[high availability]
+
+[discrete]
+[[back-end]]
+==== image:images/caution.png[with caution] back end (noun)
```

Comment:
"back end" is already covered in the IBM style guide, and we don't duplicate entries. Can you remove this?

Reply:
Done, thankyou